README.md

```diff
@@ -80,17 +80,18 @@ Fixes and optimizations are always welcome (please!), but in addition to
 that there are some major work areas still untouched:
 
 * Help work through missing functionality/bugs in game [compat](https://github.com/xenia-project/xenia/issues?labels=compat)
-* Add input drivers for [PS4 controllers](https://github.com/xenia-project/xenia/issues/60) (or anything else)
+* Add input drivers for [DS4 controllers](https://github.com/xenia-project/xenia/issues/60) (or anything else)
 * Skilled with Linux? A strong contributor is needed to [help with porting](https://github.com/xenia-project/xenia/labels/cross%20platform)
 
-See more projects [good for contributors](https://github.com/xenia-project/xenia/labels/good%20first%20issue). It's a good idea to ask on Discord/the bugs before beginning work
-on something.
+See more projects [good for contributors](https://github.com/xenia-project/xenia/labels/good%20first%20issue). It's a good idea to ask on Discord/the issues before beginning 
+work on something.
 
 ## FAQ
 
 For more see the main [frequently asked questions](https://xenia.jp/faq/) page.
 
 ### Can I get an exe?
 
-Check [Appveyor's artifacts](https://ci.appveyor.com/project/benvanik/xenia/build/artifacts) to see
-what's there.
+[Master (Vulkan)](https://ci.appveyor.com/api/projects/benvanik/xenia/artifacts/xenia-master.zip?branch=master&job=Configuration%3A%20Release&pr=false)
```

Comment:
This is a good idea, and could you change "Master (Vulkan)" -> "Master"?

Reply:
I think that was changed to match how I labeled them on the website?

Reply:
Ah, good point. I suppose we can keep it this way, though I don't like to think of branches as being tied to any one renderer backend.

Reply:
That's just a title, doesn't necessarily have to be renderer.

https://github.com/xenia-project/xenia/blob/gh-pages/_config.yml#L66

Reply:
People download from the master branch and get confused when it's not d3d12 so that's not a good idea.